package.json

```diff
@@ -83,7 +83,9 @@
     "entry:dist": "yarn lerna run prepack",
     "entry:src": "yarn lerna run postpack",
     "gas-report": "yarn workspace @latticexyz/store run gas-report && yarn workspace @latticexyz/world run gas-report",
-    "tablegen": "yarn workspace @latticexyz/store run tablegen && yarn workspace @latticexyz/world run tablegen"
+    "codegen": "yarn codegen:tablegen && yarn codegen:worldgen",
+    "codegen:tablegen": "yarn workspace @latticexyz/store run tablegen && yarn workspace @latticexyz/world run tablegen",
+    "codegen:worldgen": "yarn workspace @latticexyz/world run worldgen"
```

Comment:
This simplifies codegen to one command for CI + us, and prepares for moving these all under one CLI command